### DISCUSSION TITLE
5.0 Image size increased from 4.x

### DISCUSSION THREAD

**Discussion by ksurl** [haugene/docker-transmission-openvpn#2578] [General]
I noticed the latest version on x86 is 615mb, while the previous version was at 259mb. What is the reason for building from source instead of installing from apt? Just leftover from beta? I suspect the cruft from the build process has been left in the image.

Maybe switch back to apt install or do cleanup after build to slim it back down.

**Comment by ksurl** [comment]
adding ` && apt-get clean && rm -rf /tmp/* /var/lib/apt/lists/* /var/tmp/*` only slimmed it down to 572mb. will keep poking in the image with wagoodman/dive to find the culprit

**Reply by pkishino** [reply]
Apt doesn’t have the 4.x branch.. while there is an unofficial ppa it lacks the UI parts.. hence  build from scratch

**Reply by ksurl** [reply]
what were the issues when previously switching to alpine? the edge release has the latest released build.

**Reply by ksurl** [reply]
![image](https://user-images.githubusercontent.com/1371562/232169739-09d5d297-e730-4a70-8a48-9275f0781818.png)

so the compiled binaries are 294mb by itself. the other chunk seems to be library files. not sure there's much to trim unless some of the dependencies are unneeded for runtime.
